### ISSUE TITLE
removeFiles not working correctly

### ISSUE THREAD

**Issue by DamSenViet** [pqina/filepond#25]
Idk why but it's just not working correctly. I've tested it multiple times in the console.

removeFile() works but
removeFiles() doesn't (regardless of whether I set allowMultiple on the instance or multiple attribute on the element)

Here's the file I used to test it.

Try uploading some files into it and you can see that removeFiles() does nothing.

```html
<!doctype html>

<html>
	<head>
		<meta charset="utf-8">
		<title>test</title>
		

		<script src="filepond.js"></script>
		<link href="filepond.css" rel="stylesheet">

		<script src="filepond-plugin-image-preview.js"></script>
		<link href="filepond-plugin-image-preview.css" rel="stylesheet">

		<script src="filepond-plugin-file-encode.js"></script>
		<style>
			.filepond--root {
				max-width: 320px;
			}
		
		</style>
	</head>

	<body>
		<input type="file" multiple>
		
		
		<button>Remove All Files</button>

		<script>

		FilePond.registerPlugin(
			FilePondPluginImagePreview,
		);
		
		const inputElement = document.querySelector('input[type="file"]');
		var pond = FilePond.create(inputElement, {
			server: "https://pqina.nl/filepond/api/",
		//	allowFileEncode: true,
		//	instantUpload: false
		
		// still doesn't change anything
		//	allowMultiple: true
		});


		console.log(pond["imagePreviewMinHeight"]);
		console.log(pond["imagePreviewMaxHeight"]);
		
		
		document.querySelector('button').onclick = function(evt) {
			// removeFile works but removeFiles doesn't
			// pond.removeFile();
			pond.removeFiles();
			console.log("attempted to remove all files");
		}
		</script>
	</body>
</html>
```

**Comment by DamSenViet** [comment]
I think removeFiles() may need parameters of the file names. It probably works like processFiles() where I need to add queries otherwise it won't work.

The documentation says that removeFiles() should remove all Files and not by query.

**Comment by rikschennink** [comment]
@DamSenViet You're absolutely right, thanks for reporting!

Just published FilePond version `1.2.10` that fixes this issue.
